go.mod

```diff
@@ -475,6 +474,8 @@ require (
 	go.opentelemetry.io/proto/otlp v1.0.0 // indirect
 )
 
+require github.com/grafana/grafana-plugin-sdk-go v0.196.1-0.20231205151856-c5479f051888
```

Comment:
Using a specific commit from my [PR](https://github.com/grafana/grafana-plugin-sdk-go/pull/833) in the sdk repo. Will update to the correct sdk version after the PR in the sdk repo is merged.

Reply:
Updated to the sdk version [v0.197.0](https://github.com/grafana/grafana-plugin-sdk-go/releases/tag/v0.197.0)